data/Language/Verilog.yaml

```diff
@@ -153,15 +153,15 @@ keywords:
   - xnor
   - xor
 datatypes:
-  - shortint
-  - int
-  - longint
-  - byte
-  - bit
-  - logic
-  - reg
-  - integer
-  - real
-  - shortreal
-  - realtime
-  - time
+  - short_vlog
+  - int_c
```

Comment:
this should be `integer_vlog` ?

Reply:
https://github.com/coala/coAST/pull/63#discussion_r236746805

Reply:
You dont blindly follow anyone. :P You do the task.

your patch has a `data/DataType/integer_vlog.yaml` in it, and you have `int_c` here.

So something is wrong.

And using int_c is wrong, because Verilog has a datatype which has a keyword `integer`.  It is not identical to the C keyword which is `int`, so it can not be re-used here .

Reply:
Hmm, so he originally typed it wrong. 

I just pointed it out when he used "int" in the file thinking he would be doing it right, didn't really see it from the verilog angle.

Reply:
In SystemVerilog, integer datatypes can be classified into two broad categories, 
1. Two state integer datatypes which includes 
shortint : 16-bit signed
int : 32-bit signed
longint : 64-bit signed
byte : 8-bit signed integer
bit : User defined

2. Four state integer datatypes which includes
logic : User defined vector types.
reg : User defined vector types.
wire : User defined vector types.
integer : 32-bit signed integer.
time : 64-bit unsigned integer.

So `int_c` should be `int32` ('int32' already has a definition in https://github.com/coala/coAST/blob/master/data/DataType/int32.yaml)

Hope this helps! :)

Reply:
The int_c is renamed to int_32 now. :-P

Reply:
So the confusion between int_32(previously int_c) and integer_vlog is clear now i guess..
@siddhpant